remote-payments-tester.php

```diff
@@ -15,6 +15,8 @@ class RemotePaymentsTester {
      */
     public function __construct() {
         add_action( 'plugins_loaded', array( __CLASS__, 'init' ) );
+        add_action( 'admin_enqueue_scripts', array( __CLASS__, 'add_extension_register_script' ) );
```

Comment:
Optional, but for the sake of testing the script injection, we may want to remove this line (or we can keep it and comment out when testing).

Reply:
I'd vote commenting out in the short term since the script loading isn't quite configured/functional yet, and I'll often just be utilizing the Slotfill most likely.